ui/src/views/compute/wizard/ComputeOfferingSelection.vue

```diff
@@ -21,8 +21,7 @@
       style="width: 25vw;float: right;margin-bottom: 10px; z-index: 8"
       :placeholder="$t('label.search')"
       v-model="filter"
-      @search="handleSearch"
-      autoFocus />
```

Comment:
@utchoang could your change on autoFocus break other parts of the UI?

Reply:
@rhtyd Yes, maybe I added unnecessary autoFocus in the child components of the deploy VM form. Just delete the contents of the child components of deploy VM and it will work properly.